### DISCUSSION TITLE
Meetings conflicts with another Area

### DISCUSSION THREAD

**Discussion by dougie-wougie** [code4recovery/12-step-meeting-list#1194] [Q&A]
Josh and others:

Thank you for all the help on the way as my understanding grows using the Meeting Guide. I have an issue with Area 5 presenting Area 93 meetings about 68 of our meetings in the Meeting Guide App; more accurately, some still do not show in the Meeting Guide App for either group. We seem to null each other out. I went to Area 93 meeting today and talked to another member of District 7 (my district); he told me you might be able to help. He mentioned that the meeting address closest to the CO address would win. I read that in the docs, but that is not the case. He then mentioned that you may have the ability to make that happen. Is that possible? We are closer in location. We are [[AASCV.org](http://aascv.org/)](http://AASCV.org) in (Area 93); the other CO is [[LACCO.org](http://lacco.org/)](http://LACCO.org) (Area 5). What information do you need from me if you folks can help?

**Comment by joshreisner** [comment]
Hi @dougie-wougie 

> He mentioned that the meeting address closest to the CO address would win. I read that in the docs, but that is not the case

this is not accurate - did you say you saw some docs where it says this? if so we should get them corrected!

here are the correct docs: https://meetingguide.helpdocs.io/article/ompwn3xgy7-when-multiple-entities-list-a-meeting

reach out to the app support team [appsupport@aa.org](appsupport@aa.org) with the specific meetings and they can take a look.

**Reply by dougie-wougie** [reply]
Thank Josh, for the quick reply. 

Recapping:
On August 1, I reached out to LACCO, and they said they would not remove the meetings
On August 4, I reached out to [appsupport@aa.org](mailto:appsupport@aa.org) (see below)
On August 15, our manager at our CO went to the District meeting, deferred to the Area Meeting
On August 20, I talked to our DCM at the Area meeting and then started this discussion.

August 4th email from app support:

We understand what you are asking for, and often a re-designation of a location does and can occur in this kind of circumstance.

We ask that nearby A.A. entities work together to decide who should be the point of contact for shared locations. This includes determining how to keep listings up-to-date across multiple websites.

Generally this requires a communication protocol of some kind that is initiated when a change in an overlapping location is reported. It really is up to you all to decide how you want to handle this. Some entities choose to provide feeds to other entities. For more information on that see this support article:

[How to provide meeting listings to another entity](https://meetingguide.helpdocs.io/article/tjn0po3zco-providing-meeting-listings-to-another-entity)

Because LACOAA was the first entity to register these shared locations, they do need to be brought into the discussion if a change is to be made.   It sounds like you have reached out to them and they indicated that their policy stands in the way of this change.  It is possible that they did not clearly understand that they would still continue to list the meetings on their website.

Is it possible for  you to have another discussion with them to clarify the request?  We can provide them with more educational information as well.  The main issue is which entity will be the point of contact for this location. We will need some confirmation from them, ideally by including them on any emails back to us.

**Reply by joshreisner** [reply]
ok, glad you’ve followed the protocol.

 we don’t have anything we can do here. if gso can be persuaded to change the registration of the address(es) of the affected meetings that is historically the best route. if you can make the case that your listings for that address are better, that would help.

perhaps reps from the affected meetings can reach out to lacooa or gso to help make the case?

**Reply by dougie-wougie** [reply]
Thanks. To be honest, this is frustrating. I want to resist putting in the meeting titles (OFFICAL) and have the app will see it as a different titled meeting. Creating duplicates. Changes to a single meeting now have a duplicate effort, update the CO, and then reach out to other CO to make the same change. This will be problematic. 

D.

I will, again, go up the chain and make the case.

**Reply by joshreisner** [reply]
putting a change in the meeting title won't help. it's the physical address of the meeting that is relevant here. i really recommend against looking for a technical workaround. the solution, like they said in their email, will be better communication and coordination between the groups and their service entities.

**Reply by dougie-wougie** [reply]
I am done. Better communication did not work as me doing one on one. I will see if the Areas can work it out. I am doubtful. 

d

**Comment by dougie-wougie** [comment]
Josh:

Second part of your reply. 

You supplied this comment and link
here are the correct docs: https://meetingguide.helpdocs.io/article/ompwn3xgy7-when-multiple-entities-list-a-meeting

That document states:
It’s common for A.A. websites to have overlapping service areas. **This isn't an issue because Meeting Guide has a system for filtering out duplicates. When a website joins the app, the addresses of its meetings are all geocoded (run through Google) and then associated with that organization as the point of contact.**

Thank you. 

d

**Reply by joshreisner** [reply]
that’s correct. as you see, it’s when the meeting’s address was registered in the app, not its distance to the central office

**Reply by dougie-wougie** [reply]
So, I needed clarification. Is it the first-come? Even if they do not belong to the Area? 

D

**Reply by joshreisner** [reply]
that is correct. it's not uncommon for groups to request to be listed on multiple websites - sounds like they registered themselves with LACOAA a long time ago and your site more recently. they could ask to be removed from LACOAA?

**Reply by dougie-wougie** [reply]
Again, I did ask them to remove them; they said no. The groups may have requested to be added, or the CO just scooped them up. 

d
